contributed_definitions/NXem.nxdl.xml

```diff
@@ -171,145 +236,183 @@
+         back-scattered, electron image, diffraction pattern, EELS spectra, composition,
+         or orientation mappings to name but a few.
          
          **There are a few design choices to consider with sub-ordinate groups:**
          
-         * Above images, spectra, and mappings should be stored as NXdata instances,  
-           ideally formatted in such a way that they can be displayed with  
-           visualization software that can be specific for the file format in which  
-           the data are stored. NeXus specifies only the data model, i.e. the terms  
-           and their relations. These descriptions can be implemented and stored in  
-           JSON, HDF5, XML, or HSDS, file storage, or even other formats, although  
-           HDF5 is the most commonly used.
+         * Images and spectra should be stored as :ref:`NXimage_set` and :ref:`NXspectrum_set`  
+           instances to hold data at the earliest possible step in the computational  
+           processing (which is usually performed with the microscope control and or  
+           integrated analysis software). The formatting of the NXdata groups enables to  
+           display image and spectra with web technology visualization software.  
+           NeXus specifies only the data model, i.e. the terms and their relations.  
+           These descriptions can be implemented and stored in JSON, HDF5, XML, or HSDS,  
+           file storage, or even other formats, although HDF5 is often used.  
+         * When two- and three-dimensional geometric primitive data are stored, it is useful  
+           to write additional optional XDMF fields which support additional plotting of  
+           the data with visualization software like Paraview or Blender.  
          * Consumable results of EM characterization tasks are usually a sub-set of  
            data artifacts, as there is not an infinite amount of possible  
            electron/ion beam-specimen interactions.  
          * Images of electron counts detected in specific operation modes (bright  
-           field, dark field in TEM, secondary/back-scattered, Kikuchi in SEM)  
-         * Spectra (X-ray quanta or auger electron counts)  
-         * These data are in virtually all cases a result of some numerical
-           processing. It makes sense to name them with a controlled vocabulary,  
-           e.g. SE (secondary electron), BSE (back-scattered electron), Kikuchi,  
-           X-ray, Auger, Cathodolum(inescence) etc.  
-         
-         A key question often asked with EM experiments is how the actual (meta)data
-         should be stored (in memory or on disk). To this end the schema, here makes no
+           field, dark field in TEM, secondary/back-scattered, Kikuchi).  
+         * Spectra (X-ray quanta or Auger electron counts)  
+         * These data are in virtually all cases a result of some numerical processing. 
```

Comment:
What data?  Is this point referring to the above two points?

Reply:
yes